Apps/RaptureIntTests/src/test/java/rapture/plugin/PluginApiTest.java

```diff
@@ -81,7 +82,173 @@ public void setUp(@Optional("http://localhost:8665/rapture") String url, @Option
         pluginApi = helper.getPluginApi();
         installedSet = new HashSet<String>();
     }
+    
+    @Test(groups={"plugin","nightly"})
+    public void testInstallStructuredPlugin () {
+    	String pluginName="teststruct";
+    	String description="Test structured plugin";
+    	String zipFileName="teststructcreate.zip";
```

Comment:
Yes it's in:

https://github.com/RapturePlatform/Rapture/blob/master/Apps/RaptureIntTests/src/test/resources/plugin/nightly/teststructcreate.zip